projects/plugins/jetpack/changelog/fusion-sync-yoavf-D59840-code-1617790596

```diff
@@ -0,0 +1,3 @@
+Significance: patch
+Type: compat
+Comment: Change to post deduplication added in #19071 so that it works with posts that have gallery shortcodes. No need for a changelog entry.
```

Comment:
This seems like it should have a changelog entry to me. If nothing else because of the new "other" type for the Jetpack changelog that we're trying out in 9.7. p9dueE-2FI-p2

Reply:
This is a change for a feature that hasn't been officially announced yet. If it announced before 9.6.1 then I'm happy to change that.